### PR TITLE
[DNM] Potato Battery nerf

### PR DESCRIPTION
**Pull request by Jalleo**
Temporary values at the moment I more want to see what values maintainers want.

:cl: Jalleo
tweak: Reduces the potential battery size of potato batteries
balance:  Reduces the potential battery size of potato batteries
/:cl:

[why]: Its beyond insane that botany can make better batteries than engineering without engineering's help.

### CODE REVIEW

**Comment by kevinz000** [comment]
>nerfing potato batteries because of a golfball fuckup

hm

**Comment by Jalleo** [comment]
@kevinz000 I never even knew this was possible to be THAT much. I only just found out

**Comment by Jalleo** [comment]
Also nobody even mentioned how insane it was in the original PR.

File: code/modules/hydroponics/plant_genes.dm [hunk 1]
```diff
@@ -351,12 +351,17 @@
 			to_chat(user, "<span class='notice'>You add some cable to [G] and slide it inside the battery encasing.</span>")
 			var/obj/item/stock_parts/cell/potato/pocell = new /obj/item/stock_parts/cell/potato(user.loc)
 			pocell.icon_state = G.icon_state
-			pocell.maxcharge = G.seed.potency * 20
 
 			// The secret of potato supercells!
 			var/datum/plant_gene/trait/cell_charge/CG = G.seed.get_gene(/datum/plant_gene/trait/cell_charge)
 			if(CG) // 10x charge for deafult cell charge gene - 20 000 with 100 potency.
```
Comment (ACCount12) [inline code/modules/hydroponics/plant_genes.dm]: This is my comment, this is the value I intended. How it worked pre-nerf and post-nerf?

Comment (Jalleo) [reply code/modules/hydroponics/plant_genes.dm]: I will be updating the comment once I work out the exact numbers.

Comment (Supermichael777) [inline code/modules/hydroponics/plant_genes.dm]: Criminally wrong note btw

**Comment by Supermichael777** [comment]
400,000 seems silly holy fuck who did this

also cell charge now makes batteries worse (max 400 vs 2000) CG rate is 0.2

File: code/modules/hydroponics/plant_genes.dm (hunk 1 above)
Comment (Jalleo) [reply code/modules/hydroponics/plant_genes.dm]: Yeah like I said numbers and how it is done is up to change I dont have enough time to sort it tonight. I didnt know rate was 0.2

File: code/modules/hydroponics/plant_genes.dm [hunk 2]
```diff
@@ -351,12 +351,17 @@
...
-				pocell.maxcharge *= CG.rate*1000
+				pocell.maxcharge = CG.rate * (G.seed.potency*20)
```
Comment (Jalleo) [inline code/modules/hydroponics/plant_genes.dm]: Okay changed it about so it is less stupid since it means it actually is worse with than without.

**Comment by Jalleo** [comment]
ITT need to properly look over this tomorrow then do a clean PR coding late is bad.

**Comment by Iamgoofball** [comment]
@kevinz000 hey dipshit you realized all I did was nerf the fuck out of battery explosions right

**Pull request closed without merging**